packages/helm-charts/celo-fullnode/rc1-gcp-private-txnodes-values.yaml

```diff
@@ -1,8 +1,43 @@
+replicaCount: 10
```

Comment:
Ok sounds good. We need the `replicaCount` in the chart because we use that for generating individual pod services, but we can remove from StatefulSet when `autoscaling` is enabled.